resource/ui/ts/models/node_status.ts

```diff
@@ -141,10 +139,70 @@ module Models {
                         m("tr", [m("td", "Ranges:"), m("td", node.range_count)]),
                         m("tr", [m("td", "Leader Ranges:"), m("td", node.leader_range_count)]),
                         m("tr", [m("td", "Available Ranges:"), m("td", node.available_range_count)]),
-                        m("tr", [m("td", "Availablility:"), m("td", this._availability(nodeId))]),
+                        m("tr", [m("td", "Availablility:"), m("td", Nodes._availability(node))]),
```

Comment:
That's just from the generated code.  I just cut out a section of it and added the ellipsis myself.
this = Stores() -- when you're inside a static function.
When you're in an inside a non-static function, `this` points to the instance of the class.  You cannot access any static functions from the `this` pointer if you're inside an instance. since the functions are not on the instance.